docs/UserGuide.md

```diff
@@ -114,7 +115,8 @@ In this tutorial we will be adding your personal contact information to Reache.
 10. Now type `back` in the command box to return to the main window. Your **Main Window** should look like this:
 ![Main Window](images/Tutorial/main_window.png)
 11. Now type `view 1` in the command box to navigate to your contact details. <br><br>
-12. Use more `edit` commands to add/edit your details as you wish. When you are done, type `back` to save the data and return to the Main Window.
+12. Use more `edit` commands to edit your details as you wish. When you are done, type `back` to save the data and return to the Main Window. <br><br>
+13. As additional practice, try adding more contacts using `add` and `edit`. When you are done, type `list` to view all saved contacts.
```

Comment:
I think it would be better to add `list` in the second tutorial, since we are using `find` there?

Reply:
I though so too but listing all contacts seemed out of context for setting up and cancelling meetings. Let me try rephrasing things such that we are able to incorporate list in the second tutorial as well